src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -104,6 +104,21 @@ function transformClinicalInformationToStoreShape(patientId: string, studyId: st
         clinicalData: clinicalDataPatient
     };
     const samples = groupBySampleId(sampleIds, clinicalDataSample);
+    samples.forEach((sample, index) => {
+        let attr = _.clone(sample.clinicalData[1]);
+        attr.clinicalAttribute = {

```

Comment:
is this draft code?

Reply:
@alisman Yes, it is the dummy data I created. I will remove it later when we have the attribute in the data.

Reply:
I should add comments for the dummy data next time.